docs/docs/classic-ml/model-registry/workflow.mdx

```diff
@@ -202,6 +202,40 @@ import os
 mlflow.set_registry_uri("databricks-uc://my-databricks-shard1")
 ```
 
+#### Migrating to Databricks Unity Catalog Model Registry from Databricks Workspace Model Registry
+
+To migrate model versions from the Databricks Workspace Model Registry to the Databricks Unity Catalog Model Registry, we
+recommend using the <APILink fn="mlflow.client.MlflowClient.copy_model_version">`copy_model_version()`</APILink> API. The
+source URI would be from your workspace registry, and the destination would be a 3-level UC model name, e.g.
+
+```python
+from mlflow import MlflowClient
+
+# Registry URI must be set to workspace registry
+client = MlflowClient(registry_uri="databricks")
+src_model_uri = f"models:/my_wmr_model/1"
+uc_migrated_copy = client.copy_model_version(
+    src_model_uri, "mycatalog.myschema.my_uc_model"
```

Comment:
Does the destination model need to be created first? If so, can we clarify and show users how to create it?

Reply:
It does not actually, we'd create it if it doesn't exist already

Reply:
Got it; can we mention that in the example?